### DISCUSSION TITLE
Quality reduce?

### DISCUSSION THREAD

**Discussion by olgru** [Koenkk/zigbee2mqtt#20697] [General]
First of all: I really really really appreciate all the hard work done by Koenkk and all the supporters for delivering such a great system. I'm running HAS with MQTT and Z2M for years already. I have something around 110 Devices, running on Channel 25... Working with a Conbee2 Stick attached to a Proxmox server using a USB rj45 extension to have the stick in the ground floor while the server is downstairs.

For years the system worked pretty much stable - only 3 RGB Stripes are frequently loosing connection. Than - at a point in time, approx half a year ago - the zigbee network became unstable. Devices dropped, status of devices is not reported back, devices don't react or react delayed...
I checked Wifi network. Yes the environment is changing, neighbors start using more wifi etc. So in a first approach I added Sonoff Sticks with router firmware in the first and the second floor to improve stability. This didn't worked, the sticks loose connectivity too.
(all always up to latest FW of course)...
Using the Unify Access points in all floors I analysed the least used channels and tried them...
I replaced the USB extension cables with high quality ones - even when this worked the years before without problems.
I tried reducing the number of devices by adding a second coordinator (Skyconnect with ZHA) and move some devices to that network on channel 10. This worked, however the Z2M Conbee2 Stick still remains unstable and the devices there are still loosing connectivity.
Latest approach a week ago was to replace Conbee2 with Conbee3 - which makes the overall situation simply more worst. More devices are loosing connectivity. Devices in all floors, short or longer distance to coordinator, different vendors, grid powered or not etc...

Unfortuantelly I believe that this quality decreasing is coming from Z2M software itself. No idea if additional features, tests and routines have slowed down some processing which impacts the timing of device communication. No idea if the conitinious development has reached a point where the willingness for additional features is higher than the effort to stabilize. No idea at all, if I'm right or thinking something totally wrong. I'm simply seeing impact and thinking about potential reasons.

A mentioned at the beginning - without koenkk and all the others doing such great efforts in development, nothing would be there at all. But I'm a simple user - maybe I understand a bit about computers and coding - and I simply like to play with my home assistant smart home. But currently I'm running around to reset devices, restart the system, replug the conbee... on a daily basis. My terrasse lights sometimes don't react or switch off, the bathroom doesn't switch off the light when no one is in anymore...
No fun, no improvement, no time to play with ESP, Blutooth, MWsensors or setup more intelligent automation - because the sensors aren't stable...

I'm not willing to give up Z2M yet, as the home assistant ZHA is missing all the configuration options for the curtain movers and the temperature adjustments and all the other nice customizing configuration options. But I'm really tired in running around the whole day...

**Comment by cbundy** [comment]
If you're fairly convinced it is z2m that is having problems through updates, have you thought about simply rolling back to earlier versions?

**Comment by Ra72xx** [comment]
Welcome to the club. See https://github.com/Koenkk/zigbee2mqtt/discussions/20323 and bug https://github.com/Koenkk/zigbee2mqtt/issues/19747. 
If you can roll back, use a version not higher than 1.33.0.

**Comment by olgru** [comment]
Thanks for the comments! I really appreciate feedback.

I've switched back from ConbeeIII to the Conbee2 where I also have downgraded the Firmware to the version according to Z2M comments. Right now, I can't join any device, but everything which is connected (around 85 devices) seems to be stable. At least for the last 3 hours... ;-) Will wait and see.
This would provide a kind of evidence that Z2M have slightly problems working with the ConBee sticks. So I ordered a Zigbee/LAN device to test with other hardware...

**Reply by Ra72xx** [reply]
Well, the threads I linked are not about Conbee sticks, but more generally. All kinds of devices have problems, but especially for EZSP-based coordinators it is really hardcore annoying: Isn't ConbeeIII also EZSP-based?
